Mlem/Views/Tabs/Posts/Components/User/Components/User View.swift

```diff
@@ -46,101 +63,259 @@ struct UserView: View {
-                                .font(.caption)
-                                .foregroundColor(.secondary)
-                                .multilineTextAlignment(.center)
-                        }
-                    }
+        ScrollView {
+            CommunitySidebarHeader(
+                title: userDetails.person.displayName ?? userDetails.person.name,
+                subtitle: "@\(userDetails.person.name)@\(userDetails.person.actorId.host()!)",
+                avatarSubtext: "Joined \(userDetails.person.published.getRelativeTime(date: Date.now))",
+                bannerURL: userDetails.person.banner,
+                avatarUrl: userDetails.person.avatar,
+                label1: "\(userDetails.counts.commentCount) Comments",
+                label2: "\(userDetails.counts.postCount) Posts")
+            
+            if let bio = userDetails.person.bio {
+                MarkdownView(text: bio).padding()
+            }
+            
+            Picker(selection: $selectionSection, label: Text("Profile Section")) {
+                Text(FeedType.Overview.rawValue).tag(0)
+                Text(FeedType.Comments.rawValue).tag(1)
+                Text(FeedType.Posts.rawValue).tag(2)
+                
+                // Only show saved posts if we are
+                // browsing our own profile
+                if isShowingOwnProfile() {
+                    Text(FeedType.Saved.rawValue).tag(3)
                 }
-                .frame(maxWidth: .infinity)
+                
             }
-            .listRowBackground(Color.clear)
+            .pickerStyle(.segmented)
+            .padding(.horizontal)
             
-#warning("TODO: Make showing a user's posts and comments work")
-            /*
-             Section
-             {
-             NavigationLink {
-             ScrollView
-             {
-             LazyVStack {
-             ForEach(privatePostTracker.posts)
-             { post in
-             NavigationLink {
-             PostExpanded(account: account, postTracker: privatePostTracker, post: post, feedType: .constant(.subscribed))
-             } label: {
-             PostItem(postTracker: privatePostTracker, post: post, isExpanded: false, isInSpecificCommunity: false, account: account, feedType: .constant(.subscribed))
-             }
-             .buttonStyle(.plain)
-             }
-             }
-             }
-             .navigationTitle("Recents by \(userDetails.name)")
-             .navigationBarTitleDisplayMode(.inline)
-             } label: {
-             Text("Recent Posts")
-             }
-             
-             }
-             */
+            if selectionSection == 0 {
+                mixedFeed
+            }
+            else if selectionSection == 1 {
+                commentsFeed
+            }
+            else if selectionSection == 2 {
+                postsFeed
+            }
+            else if selectionSection == 3 {
+                savedFeed
+            }
```

Comment:
maybe a switch would be handy here instead of if else?
but it's just a nit so feel free to ignore